server/utils/forms/inputs/twelveHourClockTimeInput.test.ts

```diff
@@ -0,0 +1,200 @@
+import TwelveHourClockTimeInput from './twelveHourClockTimeInput'
+import TestUtils from '../../../../testutils/testUtils'
+import ClockTime from '../../clockTime'
+
+describe(TwelveHourClockTimeInput, () => {
+  const messages = {
+    hourEmpty: 'The alarm time must include an hour',
```

Comment:
what's 'alarm time'?

Reply:
I'm just using it as an example name of a time field, I was imaging a page for setting an alarm clock or something

Reply:
oh i see, it's supplied by the clock time class user